### PR TITLE
Update risk_methods.md

### PR DESCRIPTION
**Pull request by cronky**
Adjusted the Risk management wording - mitigation and likelihood rather than reduce and remove (in most cases its not possible to remove a risk completely).

### CODE REVIEW

File: pages/engagement_model/risk_methods.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ As stated earlier risks are all around us and organizations take calculated risk
 
 # Risk Management
 
-In order to reduce the impact or remove risks, they need to managed and monitored throughout the development of the architecture. Many of the risk management processes and techniques which are used in business and project management can be applied to managing risks in the development of an architecture both at enterprise and solution level.
+In order to mitigate the impact or reduce the likelihood of risks, they need to managed and monitored throughout the development of the architecture. Many of the risk management processes and techniques which are used in business and project management can be applied to managing risks in the development of an architecture both at enterprise and solution level.
```
Comment (paultpreiss) [inline pages/engagement_model/risk_methods.md:66]: I think this a great change.

Comment (stedo77) [reply pages/engagement_model/risk_methods.md:66]: Yes looks great!

Comment (paultpreiss) [reply pages/engagement_model/risk_methods.md:66]: Then I approve this change

File: pages/engagement_model/risk_methods.md [hunk 2]
```diff
@@ -180,4 +180,4 @@ The values for probability and consequence are used together with the risk matri
 
 ![image001](media/by-nc.png)
 
-BTABoK 3.0 by [IASA](https://iasaglobal.org/) is licensed under a [Creative Commons Attribution-NonCommercial 4.0 International License](http://creativecommons.org/licenses/by-nc/4.0/). Based on a work at [https://btabok.iasaglobal.org/](https://btabok.iasaglobal.org/)
\ No newline at end of file
+BTABoK 3.0 by [IASA](https://iasaglobal.org/) is licensed under a [Creative Commons Attribution-NonCommercial 4.0 International License](http://creativecommons.org/licenses/by-nc/4.0/). Based on a work at [https://btabok.iasaglobal.org/](https://btabok.iasaglobal.org/)
```
Comment (paultpreiss) [inline pages/engagement_model/risk_methods.md:183]: This looks good to me as well.

**Review by paultpreiss — APPROVED** [review]
Approving this

**Pull request merged**